style.css

```diff
@@ -1,134 +1,430 @@
-                cursor: pointer;
-                float: right;
-            }
-            
-            .infoButton:hover {
-                background-color: #e7e7e7;
-            }
-            
-            .infoBox {
-                height: 550px;
-                width: 700px;
-                background-image: linear-gradient(to right top, #7bd573, #55c079, #33aa7a, #139378, #007d71, #00766f, #006f6c, #056869, #046e6f, #037474, #017a7a, #008080);
-                float: left;
-            }
-            
-            .image {
-                width: 210px;
-                height: 240px;
-                image-rendering: crisp-edges;
-                border: 6px groove #e7e7e7;
-                margin: 8px;
-                background-color: #066061;
-            }
-            
-            .triangleLeft {
-                width: 0;
-                height: 0;
-                border-top: 25px solid transparent;
-                border-right: 50px solid #7bd573;
-                border-bottom: 25px solid transparent;
-                display: inline-block;
-                position: absolute;
-                left: 30px;
-                bottom: 0;
-            }
-            
-            .triangleRight {
-                width: 0;
-                height: 0;
-                border-top: 25px solid transparent;
-                border-left: 50px solid #7bd573;
-                border-bottom: 25px solid transparent;
-                right: 30px;
-                bottom: 0;
-                display: inline-block;
-                position: absolute;
-            }
-            
-            .infoList {
-                list-style-type: none;
-                padding-right: 100px;
-                margin-right: 30px;
-                margin-top: 50px;
-            }
-            
-            .pokeImg {
-                float: left;
-                height: 320px;
-                position: relative;
-            }
-            
-            .divInfoList {
-                float: right;
-                height: auto;
-                width: auto;
-                font-size: 30px;
-                text-align: right;
-            }
-            
-            .infoName {
-                height: 230px;
-                width: auto;
-                ;
-            }
-            
-            .pokeNameParagraph {
-                margin-left: 35px;
-                padding-left: 60px;
-                padding-top: 60px;
-                font-size: 44px;
-                float: right;
-                margin-right: 130px;
-            }
-            
-            .displayDateAndTimeParagraph {
-                top: 520px;
-                left: 820px;
-                position: fixed;
-                width: 190px;
-                height: 40px;
-                color: #7bd573;
-            }
\ No newline at end of file
+                @use postcss-color-function;
+                @use postcss-nested;
```

Comment:
Is these `@use' necessary? What are these?

Reply:
Fixed